### DISCUSSION TITLE
I am planning to change the look and feel of web app.

### DISCUSSION THREAD

**Discussion by ashishonmobile** [alfio-event/alf.io#1213] [Ideas]
Hello Guys,

I am planning to apply few modern theme for webapp. I am very new to this project and still trying to understand it.

Anybody has any guidance for changing the theme for the project will be really helpful.

Regards,
Ashish

**Comment by syjer** [comment]
hi @ashishonmobile , the simplest way to customize the css is to go in the administration under the "Configuration" tab.

There is a textarea (we know, kinda rough, but are redoing the whole admin. To be noted, you can customize at "system", "organization" or even "event" level):

![customize_css](https://user-images.githubusercontent.com/498146/235774116-b8764807-5105-4c90-b9af-8a3f5a062c51.png)

In it you can add anything you want as a css. The simplest way would be to add an `@import` rule ( https://developer.mozilla.org/en-US/docs/Web/CSS/@import ) to an external css and modify it there for a more immediate feedback.

We will document it better about this possibility.
